### PR TITLE
feat(flush stats): flush stats method

### PR DESCRIPTION
**Pull request by sujeshthekkepatt**
Currently no method to flush the stats.
Flushing stats helpful for metrics analysis. See more #158

### CODE REVIEW

**Comment by sujeshthekkepatt** [comment]
@erdii please take a look

File: _src/lib/node_cache.coffee [hunk 1]
```diff
@@ -489,6 +489,38 @@ module.exports = class NodeCache extends EventEmitter
...
+	
+		# ## flushStats
+	#
+	# flush the stats
+	#
+	# **Example:**
+	#
+	#     myCache.flushStats()
+	#
+	#     myCache.flushStats()
```
Comment (erdii) [inline _src/lib/node_cache.coffee]: don't you mean `myCache.getStats()` here?

File: _src/test/mocha_test.coffee [hunk 2]
```diff
@@ -661,7 +661,28 @@ describe "`#{pkg.name}@#{pkg.version}` on `node@#{process.version}`", () ->
...
+			value = randomString 10
+			res = cache.set key,value
+			true.should.eql res
+			1.should.eql cache.getStats().keys
+			cache.flushStats()
+			0.should.eql cache.getStats().keys
+			cache.get key
+			1.should.eql cache.getStats().hits
+			cache.get randomString 10
+			1.should.eql cache.getStats().misses
```
Comment (erdii) [inline _src/test/mocha_test.coffee]: i think the actual call to cache.flushStats() and the additional test if everything has been flushed are missing here

**Comment by erdii** [comment]
wait nevermind! i'll just merge it and add the changes myself. Then I can create a new release with t his and .take() from #160

File: _src/test/mocha_test.coffee (hunk 2 above)
Comment (erdii) [reply _src/test/mocha_test.coffee]: oh wait. I just missed the flushStats() call here :see_no_evil:

**Pull request merged**